mtags/src/main/scala/scala/meta/internal/mtags/DefinitionAlternatives.scala

```diff
@@ -33,6 +34,19 @@ object DefinitionAlternatives {
       GlobalSymbol(owner, Descriptor.Type(name))
     }
 
+  private val objectMethods = Set("equals", "toString", "getClass", "hashCode")
+
+  /**
+   * If Any is used we don't have any source to go to, so instead we try with java Object.
+   */
+  private def objectInsteadOfAny(symbol: Symbol): Option[Symbol] = {
```

Comment:
It seems a bit misleading, though I guess it is reasonable enough.

Reply:
Any ideas for an alternative name? I can't come up with anything clearer :sweat: